sqlalchemy_redshift/dialect.py

```diff
@@ -604,6 +616,11 @@ def _get_table_or_view_names(self, relkind, connection, schema=None, **kw):
     def _get_column_info(self, *args, **kwargs):
         kw = kwargs.copy()
         encode = kw.pop('encode', None)
+        if 'comment' in arglist(super(RedshiftDialect, self)._get_column_info):
```

Comment:
if we're worried about perf, how about a global cache of:

```python
has_comment = 'comment' in arglist(PGDialect_psycopg2._get_column_info)
```

Reply:
I'm leaning toward reverting to version check at this point. I don't see a clear benefit to doing the more complicated thing.